ce/ce/constants.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const project = 'environment.yaml';
+export const undo = 'Z_VCPKG_UNDO';
+export const postscriptVarible = 'Z_VCPKG_POSTSCRIPT';
+export const blank = '\n';
+export const gitUniqueIdPrefix = 'https://aka.ms/vcpkg-ce-default::tools/git::';
+export const gitArtifact = 'microsoft:tools/git';
+export const latestVersion = '*';
+export const vcpkgDownloadFolder = 'VCPKG_DOWNLOADS';
+export const globalConfigurationFile = 'vcpkg-configuration.global.json';
+export const profileNames = ['vcpkg-configuration.json', 'vcpkg-configuration.yaml', 'environment.yaml', 'environment.yml', 'environment.json'];
```

Comment:
I believe vcpkg-configuration.json is the only supported filename.

Reply:
I really don't want to make functionality changes like that in this PR. I think the search behavior being wrong is already tracked by https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1494960